src/controller/CHIPDeviceController.cpp

```diff
@@ -220,23 +218,26 @@ void DeviceController::OnLocalNOCChainGeneration(void * context, CHIP_ERROR stat
 
     DeviceController * controller = static_cast<DeviceController *>(context);
 
-    Transport::FabricInfo * const fabric = controller->mFabrics.FindFabricWithIndex(controller->mFabricIndex);
+    Transport::FabricInfo newFabric;
+
+    newFabric.SetOperationalKey(&controller->mOperationalKey);
```

Comment:
This reads oddly: a fabric does not have an operational key. Not sure what is intended here

Reply:
It's ephemeral key used for CSR generation. I'll rename it.